### PR TITLE
Add volume mounts having CA Certs for the pods

### PR DESCRIPTION
**Pull request by ajaysoni-cap**
# **What this PR does / why we need it**:
Add volume mounts on the PODS having CA Certs.
Relates to ADO Work Item [AB#261145](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/261145)

# **Special notes for your reviewer**
N/A

# Testing
N/A

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
TomiTominetGIF

### CODE REVIEW

File: adp-helm-library/templates/_container.yaml [hunk 1]
```diff
@@ -24,6 +24,10 @@ resources:
 volumeMounts:
 - mountPath: /tmp
   name: temp-dir
+- name: custom-root-ca
+  mountPath: /etc/ssl/certs/defra-cert-01.crt
```
Comment (DaRoszko) [inline adp-helm-library/templates/_container.yaml]: defra-egress-fw-cert ?

**Review by DaRoszko — CHANGES_REQUESTED** [review]

File: adp-helm-library/templates/_container.yaml (hunk 1 above)
Comment (DaRoszko) [reply adp-helm-library/templates/_container.yaml]: and 01 if you want?

Comment (DaRoszko) [reply adp-helm-library/templates/_container.yaml]: or firewall instead of fw. gives strong clarity ?

Comment (ajaysoni-cap) [reply adp-helm-library/templates/_container.yaml]: Changed the name.

**Review by DaRoszko — APPROVED** [review]

**Review by ppotturi — APPROVED** [review]

**Pull request merged**